### PR TITLE
Initial work toward i18n for loadout builder

### PR DESCRIPTION
**Pull request by ericnelson0**
Issue #899 

I think this is all of the text that I can take from the manifest.

image

Does anyone see any more areas that could be changed?

### CODE REVIEW

**Comment by delphiactual** [comment]
`Filter Sets (Intellect/Discipline/Strength)` - inside parenthesis can be translated, from our i18n file.
`Create Loadout` can also be translated from same source.
And copy that is too long (e.g. `Objetos de clase`) should be wrapped so that the spacing between items remains consistent.

**Comment by ericnelson0** [comment]
@delphiactual Thanks.

Still a bunch of our own strings that need translation.

**Comment by bhollis** [comment]
What's left on this?

**Comment by ericnelson0** [comment]
Still a lot of our strings that need to be translated. Not sure if that needs to be done first.

**Comment by delphiactual** [comment]
@ericnelson0 `Armor` is already translated. It is used before the dropdown menu, you'll just have to concat the `:`

**Comment by delphiactual** [comment]
https://github.com/DestinyItemManager/DIM/pull/1071

**Comment by delphiactual** [comment]
@ericnelson0 the translate strings you used are incorrect you should be using the CamelCased strings names (e.g. BackToDIM not "Back To DIM")

**Comment by ericnelson0** [comment]
@delphiactual I'm not sure if we want to change that to remove a number completely, but it is being calculated as 350.

**Comment by delphiactual** [comment]
updated copy to specify 350

**Comment by delphiactual** [comment]
@ericnelson0 Once this and https://github.com/DestinyItemManager/DIM/pull/1072 are merged I will complete the i18n for Loadout Builder

**Comment by delphiactual** [comment]
@ericnelson0 sent you a PR for translations, this PR now depends on https://github.com/DestinyItemManager/DIM/pull/1072 for the translation values that it uses.

**Comment by ericnelson0** [comment]
Ok, I'll take a look tonight.

File: app/views/best.html [hunk 1]
```diff
@@ -32,7 +32,11 @@ <h1 translate="LB"></h1>
   </div>
 
   <div class="section">
-    <p><span translate="LB.Locked"></span> - <small translate="LB.LockedHelp"></small></p>
+    <p>
+      <span class='dim-button locked-button' ng-click="normal.lockEquipped()">Lock Equipped</span>
```
Comment (delphiactual) [inline app/views/best.html]: `<span class='dim-button locked-button' ng-click="normal.lockEquipped()" translate>LB.LockEquipped</span>`

File: app/views/best.html [hunk 2]
```diff
@@ -32,7 +32,11 @@ <h1 translate="LB"></h1>
...
+      <span class='dim-button locked-button' ng-click="normal.clearLocked()">Clear Locked</span>
```
Comment (delphiactual) [inline app/views/best.html]: `<span class='dim-button locked-button' ng-click="normal.clearLocked()" translate>LB.ClearLocked</span>`

File: app/views/best.html [hunk 3]
```diff
@@ -32,10 +32,13 @@
   </div>
 
   <div class="section">
-    <p><span class='dim-button locked-button' ng-click="normal.lockEquipped()">Lock Equipped</span>
-       <span class='dim-button locked-button' ng-click="normal.clearLocked()">Clear Locked</span>
-       Locked Items <small>- Drag and drop any item into its bucket to build sets with that specific gear. <span ng-show="!normal.excludeditems.length">Shift + click to exclude items.</span></small></p>
+    <p>
+      <span class='dim-button locked-button' ng-click="normal.lockEquipped()">Lock Equipped</span>
```
Comment (delphiactual) [inline app/views/best.html]: `<span class='dim-button locked-button' ng-click="normal.lockEquipped()" translate="LB.LockEquipped"></span>`

File: app/views/best.html [hunk 4]
```diff
@@ -32,10 +32,13 @@
...
+      <span class='dim-button locked-button' ng-click="normal.clearLocked()">Clear Locked</span>
```
Comment (delphiactual) [inline app/views/best.html]: `<span class='dim-button locked-button' ng-click="normal.clearLocked()" translate="LB.ClearLocked"></span>`

**Comment by ericnelson0** [comment]
@delphiactual I translated stat type string in the popup over the stats.

Do you want to keep this open for a while in order to get some translation for other languages?

**Comment by delphiactual** [comment]
@ericnelson0 no, just go ahead and merge

**Comment by ericnelson0** [comment]
To anyone wondering, this is basically ready to go, but I want to do some testing over the weekend to sniff out any bugs. Should be merged by the beginning of next week

**Comment by kyleshay** [comment]
I'll merge it now so it can be tested in the beta branch, thanks @ericnelson0 !

**Comment by kyleshay** [comment]
Also, please close any other issues that this PR may have resolved.

**Pull request merged**